.github/workflows/oma-dev-env.yml

```diff
@@ -0,0 +1,29 @@
+name: "Update cachix cache for Orchestrating Microbiome Analysis environment"
+
+on:
+  push:
+    branches: [ devel, nix ]
```

Comment:
I think we could remove "nix" branch from here (just make sure that the run works first; it is failing now)

Reply:
To make the tree tidier, there should be only devel, and gh-pages branch. Other branches are temporary

Reply:
--> which means that nix branch will be deleted. (if it is not required)

Reply:
It's failing because I cannot add `CACHIX_AUTH` variable that's required to push to Cachix.